src/Hero.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+const Section = styled.section`
+  text-align: center;
+  margin-top: 51px;
+`;
+
+const Title = styled.h1`
+  font-family: 'Bitter', serif;
+  font-weight: normal;
+  font-size: 38px;
+  letter-spacing: 0.8px;
+`;
+
+const Subtitle = styled.p`
+  color: #93918f;
+`;
+
+const Button = styled.button`
+  width: 217px;
+  height: 36px;
+  border-radius: 4px;
+  border: none;
+  background-color: #fdb755;
+  color: #fff;
+  text-transform: uppercase;
+  font-size: 14px;
+  letter-spacing: 0.5px;
+  margin-top: 30px;
+  cursor: pointer;
+`;
+
+const StyledSub = styled.p`
+  margin-top: 45px;
+  color: #93918f;
```

Comment:
This color could be defined globally as well. It might also be defined in the `theme` (if you have one of course 😄 )